### PR TITLE
[#9] Run rubocop in CI

### LINKED ISSUE #9: Run rubocop in CI
Rubocop seems to not be run on Travis CI at the moment.

### PR DESCRIPTION
**Pull request by tessi**
**TL;DR**

* attempt to fix #9 
* apply CI hints Andy gave in his last lunch talk
* test the latest ruby version
* require a minimal ruby version of `2.1` (from `2.0`)
* track `Gemfile.lock` in git

**Longer version**

Different rubocop-versions have different cops (and, thus, require different code formattings). This is why we need to have the same rubocop version for all ruby-version builds. To achieve that, I removed `Gemfile.lock` from `.gitignore`. With a `Gemfile.lock` in version control, we check against the same rubocop configuration for all builds.

Next task is to find a set of gems working on all supported ruby versions. Because of the ruby-2.0.0 error

```
rubocop-rspec-1.13.0 requires ruby version >= 2.1.0, which is incompatible with
```

I decided to require a minimal ruby version of `2.1` (instead of downgrading rubocop-rspec). Hope that's OK with everyone here.

### CODE REVIEW

**Review by andreasknoepfle — APPROVED** [review]
Great ❤️ . Love it.
